apps/worker/package.json

```diff
@@ -13,7 +13,7 @@
     "docker:build:depot": "pnpm --silent --workspace-root pnpm-context -- apps/worker/Dockerfile | depot build --build-arg PACKAGE_PATH=apps/worker - -t novu-worker --load",
     "start": "pnpm start:dev",
     "start:dev": "nest start --watch",
-    "start:test": "cross-env NODE_ENV=test nest start",
+    "start:test": "cross-env IS_WORKFLOW_PREFERENCES_ENABLED=true NODE_ENV=test nest start",
```

Comment:
I could drop this into `.env.test` in Worker instead, I agree it could get unwieldy. I'll do that instead.

The line suggested doesn't work because we are running the tests that depend on this env-var being set from API, rather than Worker.